string camera_id

int32 detection

float32 confidence
